app/Config/Constants.php

```diff
@@ -38,9 +38,9 @@
 defined('HOUR')   || define('HOUR', 3600);
 defined('DAY')    || define('DAY', 86400);
 defined('WEEK')   || define('WEEK', 604800);
-defined('MONTH')  || define('MONTH', 2592000);
-defined('YEAR')   || define('YEAR', 31536000);
-defined('DECADE') || define('DECADE', 315360000);
+defined('MONTH')  || define('MONTH', 2_592_000);
+defined('YEAR')   || define('YEAR', 31_536_000);
+defined('DECADE') || define('DECADE', 315_360_000);
```

Comment:
What is this black magic?

Reply:
It generated by 

https://github.com/rectorphp/rector-src/blob/1e3e5dff59b05d497580f52e9a7491a2afa6bd29/rules/Php74/Rector/LNumber/AddLiteralSeparatorToNumberRector.php#L30

We can skip the rule if we want to exclude it

Reply:
I'm not aware this was possible with PHP 7.4. Nice 👍

Reply:
Yes, I like it as well! I didn't know that it was possible was all. Keep!